src/index.html

```diff
@@ -15,8 +15,90 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#">
+        <img
+          src="./images/logo.png"
+          class="logo"
+          alt="logo"
```

Comment:
The `alt` attribute should describe the image content. Instead of 'logo', it would be better to write more descriptive information about the image, for example, the name of the company or website the logo belongs to.